src/node_worker.cc

```diff
@@ -897,6 +900,142 @@ void Worker::CpuUsage(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+class WorkerCpuProfileTaker : public AsyncWrap {
+ public:
+  WorkerCpuProfileTaker(Environment* env, Local<Object> obj)
+      : AsyncWrap(env, obj, AsyncWrap::PROVIDER_WORKERCPUPROFILE) {}
+
+  SET_NO_MEMORY_INFO()
+  SET_MEMORY_INFO_NAME(WorkerCpuProfileTaker)
+  SET_SELF_SIZE(WorkerCpuProfileTaker)
+};
+
+void Worker::StartCpuProfile(const FunctionCallbackInfo<Value>& args) {
+  Worker* w;
+  ASSIGN_OR_RETURN_UNWRAP(&w, args.This());
+  Environment* env = w->env();
+
+  CHECK(args[0]->IsString());
+  node::Utf8Value name(env->isolate(), args[0]);
+
+  AsyncHooks::DefaultTriggerAsyncIdScope trigger_id_scope(w);
+  Local<Object> wrap;
+  if (!env->worker_cpu_profile_taker_template()
+           ->NewInstance(env->context())
+           .ToLocal(&wrap)) {
+    return;
+  }
+
+  BaseObjectPtr<WorkerCpuProfileTaker> taker =
+      MakeDetachedBaseObject<WorkerCpuProfileTaker>(env, wrap);
+
+  bool scheduled = w->RequestInterrupt([taker = std::move(taker),
+                                        name = name.ToString(),
+                                        env](Environment* worker_env) mutable {
+    Local<Value> title;
+    if (!ToV8Value(worker_env->context(), name.data(), worker_env->isolate())
+             .ToLocal(&title)) {
+      return;
+    }
+    CpuProfilingResult result = worker_env->StartCpuProfile(title.As<String>());
```

Comment:
Here we convert back and forth between `std::string` and `v8::Local<v8::String>`, maybe `StartCpuProfile()` and `StopCpuProfile()` should just be two overloads, where one takes a `std::string_view` and the other a `v8::Local<v8::String>`?

Reply:
Have updated `v8::Local<v8::String>` to `std::string_view`.